fast64_internal/oot/actor/properties.py

```diff
@@ -442,36 +444,40 @@ def draw_props(self, layout: UILayout, altRoomProp: OOTAlternateRoomHeaderProper
         split.label(text="Actor ID")
         split.label(text=getEnumName(ootData.actorData.ootEnumActorID, self.actor_id))
 
-        if self.actor_id != "Custom":
-            self.draw_params(actorIDBox, obj)
-        else:
-            prop_split(actorIDBox, self, "actor_id_custom", "")
+        if bpy.context.scene.fast64.oot.use_new_actor_panel:
+            if self.actor_id != "Custom":
+                self.draw_params(actorIDBox, obj)
+            else:
+                prop_split(actorIDBox, self, "actor_id_custom", "")
```

Comment:
The custom id is being hidden but still being exported?

Reply:
I'm not sure what you mean in both cases, all this does is display the custom string prop if the custom id was chosen (otherwise draw the param's ui for that actor)

Reply:
Yes, but the custom id does not show up under the old actor panel yet is still exported, that's what I was commenting on.

Reply:
Yeah as lila is saying this change means you can't set a custom actor id using "old actor panel"
![image](https://github.com/user-attachments/assets/b114567a-ab60-408d-a8b2-f4639eea0c22)
Not sure what you intended

Reply:
should be fixed now